compiler/damlc/tests/daml-test-files/InterfaceChoiceCollision.daml

```diff
@@ -7,7 +7,6 @@
 -- This coverts also regression test for implements and method markers.
 --  See https://github.com/digital-asset/daml/issues/13915
 
```

Comment:
I think we should add a comment to make clear what this test does now

```suggestion

-- Note that there is no error here even though InterfaceChoiceCollision1.Interface and
-- InterfaceChoiceCollision2.Interface both define choices named MyArchive. This is 
-- because the interface type is used to disambiguate between them.
```